packages/react-dom/src/test-utils/ReactTestUtilsAct.js

```diff
@@ -31,9 +31,12 @@ const [
   runEventsInBatch,
   /* eslint-enable no-unused-vars */
   flushPassiveEffects,
-  IsThisRendererActing,
 ] = ReactDOM.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.Events;
 
+const [
+  IsThisRendererActing,
+] = ReactDOM.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.Tests;
```

Comment:
How about we just don't destructure nor set `IsThisRendererActing` in ReactTestUtilsAct for PROD?

It's only ever read by React in DEV mode so why bother writing it in PROD? Even if we do use the rest of act in prod.